capsules/core/src/driver.rs

```diff
@@ -97,5 +97,6 @@ pub enum NUM {
     DateTime              = 0x90007,
     CycleCount            = 0x90008,
     Servo                 = 0x90009,
+    EthernetTap           = 0x90010,
```

Comment:
Not sure whether this is the right driver category. Should we perhaps rename "Radio" to "Networking" or "Communication"? I don't know which category is best suited for this and other non-radio network components.

Reply:
Yes, I think we should rename "Radio" to "Networking" and add this to that section